pom.xml

```diff
@@ -1195,6 +1196,8 @@
                 <module>persistence-modules/questdb</module>
                 <module>vaadin</module>
                 <module>libraries-3</module>
+                <module>web-modules/apache-tapestry</module>
+                <!--<module>web-modules/apache-tapestry</module>-->
```

Comment:
add the entire web-modules here and then in the pom.xml of web-modules enable those that. you have upgraded for e.g. apache-tapestry and comment out the rest.

Reply:
sure but I have divided the task into sub-tasks. Should I add all the rest modules in their jira commit?